R/ggsurvplot.R

```diff
@@ -303,6 +312,7 @@ ggsurvplot <- function(fit, fun = NULL,
   if(is.null(ylim) & is.null(fun)) ylim <- c(0, 1)
   if(!is(legend, "numeric")) legend <- match.arg(legend)
   surv.median.line <- match.arg(surv.median.line)
+  stopifnot(log.rank.weights %in% c("survdiff", "1", "n", "sqrtN", "S1", "S2","FH_p=1_q=1"))
```

Comment:
How to define another weight function? Can this vector of weights be edited and made larger? If yes, how?